common/common-rest/src/main/java/org/apache/servicecomb/common/rest/resource/ClassPathStaticResourceHandler.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.servicecomb.common.rest.resource;
+
+import java.io.IOException;
+import java.net.URL;
+
+import javax.servlet.http.Part;
+
+import org.apache.servicecomb.foundation.common.part.InputStreamPart;
+
+public class ClassPathStaticResourceHandler extends StaticResourceHandler {
+  protected Part findResource(String path) throws IOException {
+    URL url = this.getClass().getClassLoader().getResource(path);
```

Comment:
Is it better to constraint the static contents are located in $jar/webapp ? Or we need to check the file suffix to make sure not download unexpected files.

Reply:
already checked in:
org.apache.servicecomb.common.rest.resource.StaticResourceHandler#handle

findResource is a protected method

Reply:
ok, is it better to rename webroot to static? So that we can migrate from spring boot. Or we can add a configuration for webroot.

Reply:
ResourceHandler is a reuseable class, it's not provide service directly
if someone want to change the directory, can do it manually, eg: 
in org.apache.servicecomb.inspector.internal.InspectorImpl, invoke: resourceHandler.setWebRoot("static/")